src/components/DailyTasks.js

```diff
@@ -1,48 +1,58 @@
-import {useEffect, useState} from 'react';
+import {useState, useEffect} from 'react';
+import {dailyTodos} from '../db';
+
 import {loadFromLocalStorage, writeToLocalStorage} from '../util/localstorage';
-import styled from 'styled-components';
 import TaskList from './TaskList';
-import dailyTasks from '../db';
 
 export default function DailyTasks() {
   const [todos, setTodos] = useState(() => {
     const todosFromLocal = loadFromLocalStorage('todos');
-    const uncheckedTodos = todosFromLocal?.map(todo =>
-      todo.checkedAt !== new Date().toLocaleDateString() ? {...todo, isChecked: false, checkedAt: ''} : todo
-    );
-    return uncheckedTodos ?? dailyTasks;
+    const uncheckedTodos = todosFromLocal?.map(todo => {
+      if (todo.tasks !== undefined) {
```

Comment:
@stefankorder 
I check here if "tasks" is present in the object, because there are no tasks for the weekend object ("samstags und sonntags"), only a picture. That's why we can't map over todo.tasks. Is there a nicer solution? :)